save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/GithubLocation.kt

```diff
@@ -0,0 +1,36 @@
+package com.saveourtool.save.entities
+
+import javax.persistence.Entity
+
+/**
+ * Entity class with github repository information
+ * @property organization
+ * @property repository
+ * @property token it can be null for public repositories
+ * @property branch a specific branch name with test, otherwise default branch (main or master)
+ * @property subDirectory a path to test suites in repository, otherwise root directory
+ */
+@Entity
+class GithubLocation(
```

Comment:
I think we should support multiple git providers from the very beginning. Maybe we can make something like `GitRemoteLocation` with implementations for different services? I think the only difference would be in how URL is constructed based on repo name and organization name

Reply:
> I think we should support multiple git providers from the very beginning.
GithubLocation is extracted for this purpose on future, I thought about interface `Location`, which encapsulates location of tests (GIT, SVN and etc.). Now it's not required and I added only `GithubLocation`

> I think the only difference would be in how URL is constructed based on repo name and organization name
also credentials provider -- GitHub supports `token` without user\password.